app/assets/javascripts/rails_admin/ra.filtering-multiselect.js

```diff
@@ -281,7 +317,9 @@
     },
 
     selected: function(value) {
-      return this.element.find('option[value="' + value + '"]').attr("selected");
+      if (this.selection[0].querySelectorAll('option[value="' + value + '"]')[0]) {
+        return true;
+      }
```

Comment:
its merged like that in main rails admin project. i would keep it so we have consistent code